Makefile

```diff
@@ -55,6 +55,25 @@ $(OUT)/emulate.o: CFLAGS += -fno-gcse -fno-crossjumping
 endif
 endif
 
+MINI_GDBSTUB :=
+ENABLE_GDBSTUB ?= 1
+ifeq ("$(ENABLE_GDBSTUB)", "1")
+MINI_GDBSTUB_OUT = $(PWD)/build/mini-gdbstub
+CONFIG_SOCKET_ADDR = 127.0.0.1:1234
```

Comment:
In GDB, the combination of host and port number is called "comm": https://sourceware.org/gdb/onlinedocs/gdb/Server.html
You can say, `GDBSTUB_COMM`.